tsconfig.json

```diff
@@ -18,6 +18,14 @@
     "lib": [
       "es2018",
       "dom"
-    ]
+    ],
+    "paths": {
+      "@ngxs-labs/entity-state": [
```

Comment:
does that make sense?

Reply:
All tsconfig files extends this file, so why not?

Reply:
I mean, how much is the difference in use?

"@ngxs-labs/entity-state" instead "@ngxs-labs/entity-state/*"

Reply:
never seen this `"@ngxs-labs/entity-state":`, but only `"@ngxs-labs/entity-state/*":`

Reply:
then maybe it's worth removing it?

Reply:
I don't know what the first does, if removing it results in no error then I'd say yes.

Reply:
I copied that from other lab's project.
I'm checking now if it's necessary.

Reply:
@splincode @odahcam
That's not working, import using `"@ngxs-labs/entity-state/*"` will look like that:
```typescript
import { ... } from '@ngxs-labs/entity-state/index';
```

Reply:
@eranshmil I'm not an expert here, but I would say do what works and what exposes the minimal internal api possible. 👍